### PR TITLE
[137] - Very Big informer

### PR DESCRIPTION
**Pull request by Andyatkov**
Simulator Screen Shot - iPhone 14 Pro - 2022-12-19 at 20 43 01

### CODE REVIEW

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 28
Version: 1671471903
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/137-big-informer
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/28

File: lib/src/widgets/views/informers/big_informer.dart [hunk 1]
```diff
@@ -0,0 +1,112 @@
+import 'package:admiralui_flutter/admiralui_flutter.dart';
+import 'package:admiralui_flutter/layout/layout_grid.dart';
+import 'package:flutter/material.dart';
+
+// BigInformer - The component is in four status colors, does not have a Pop-up.
+// It consists of several elements that can be disabled in layers.
+// Adapts to the width of the device screen. The maximum height
+// of the component is 400 px.
+class BigInformerWidget extends StatefulWidget {
```
Comment (timbaton) [inline lib/src/widgets/views/informers/big_informer.dart:9]: почему Stateful

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 32
Version: 1671532449
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/137-big-informer
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/32

**Review by timbaton — APPROVED** [review]

**Review by KirBorisov96 — APPROVED** [review]

**Comment by KirBorisov96** [comment]
Platform: iOS
Build ID: 33
Version: 1671533460
Internal Version: 0
Short Version: 1.0.0
Branch Name: feature/137-big-informer
Install URL: https://install.appcenter.ms/orgs/Admiral-Digital-Frameworks-App/apps/AdmiralUI-Flutter-iOS-Dev/releases/33

**Pull request merged**